keras/backend/cntk_backend.py

```diff
@@ -1649,17 +1652,18 @@ def conv3d(x, kernel, strides=(1, 1, 1), padding='valid',
     x = _preprocess_conv3d_input(x, data_format)
     kernel = _preprocess_conv3d_kernel(kernel, data_format)
     padding = _preprocess_border_mode(padding)
-    strides = strides + (strides[0],)
```

Comment:
Here is bug, as even we build a normal convolution(non-dilated) layer, it gets different output compared with TF and TH backend. After the fix, it produces same output.